src/stories/components/Section/styles.scss

```diff
@@ -0,0 +1,9 @@
+@import '../../../scss/variables.scss';
+
+.section--medium {
+    font-size: $font-size-lg
```

Comment:
Default font-size is "$font-size-md". 

In order to create a "default" / medium / large, I have opted to use the following scale for font sizes:
- default section = $font-size-md (16px)
- medium section = $font-size-lg (20px)
- large section = $font-size-xl (30px)